src/neo-vm/ExecutionEngine.cs

```diff
@@ -1207,6 +1264,56 @@ private bool ExecuteStoreToSlot(Slot slot, int index)
             return true;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private bool ExecuteTry(int catchOffset, int finallyOffset)
+        {
+            if (catchOffset == 0 && finallyOffset == 0) return false;
+            int catchPointer = catchOffset == 0 ? -1 : checked(CurrentContext.InstructionPointer + catchOffset);
+            int finallyPointer = finallyOffset == 0 ? -1 : checked(CurrentContext.InstructionPointer + finallyOffset);
+            CurrentContext.TryStack ??= new Stack<ExceptionHandingContext>();
+            CurrentContext.TryStack.Push(new ExceptionHandingContext(catchPointer, finallyPointer));
+            CurrentContext.MoveNext();
```

Comment:
`MoveNext` returns is never used, maybe we should change it to `void`

Reply:
I think we can change it.

Reply:
Let's keep it.